CONTRIBUTING.md

```diff
@@ -241,23 +241,23 @@ git checkout dev
 
 ### Installing Dart
 
-This project requires using Dart version **3.5.4**, not the latest version. Click on a dropdown below for installation instructions for your operating system.
+This project requires using Dart version **3.7.3**, not the latest version. Click on a dropdown below for installation instructions for your operating system.
```

Comment:
Let's change this. These instructions were for when we did not have null safety. Now I'd rather if the instructions are always to use the latest Dart. If we run into another problem where we don't seem to be able to upgrade, I'll revise these instructions. But for now please write it to indicate how to get the latest version of dart. But leave these instructions as a separate subsection later titled "Installing a particular version of dart".